lib/id3c/cli/redcap.py

```diff
@@ -369,6 +369,50 @@ def metadata(self) -> List[Dict[str, str]]:
         return self._fetch('metadata', {})
 
 
+    def update_metadata(self, metadata: Dict[str, str]) -> int:
+        """
+        Update existing *metadata* in this REDCap project.
+
+        *metadata* must be an iterable of :py:class:``dict``s mapping REDCap
+        field names, form names, and other instrument metadata to values.  The
+        instrument field name, form name, and field type, and field label, at a
+        minimum, must be included.   All keys and values must be strings.
+
+        Any value provided for a field, including the empty string, will
+        overwrite any existing value. From the REDCap API Documentation:
+
+        > Because of this method's destructive nature, it is only available for
+        > use for projects in Development status.
+
+        Returns a count of the number of records updated, as reported by
+        REDCap.
+        """
+        parameters = {
+            'data': json.dumps(metadata),
+            'type': 'flat',
+            'overwriteBehavior': 'overwrite',
+            'returnContent': 'count',
+        }
+
+        expected_count = len(metadata)
+
+        if not self.dry_run:
+            LOG.debug(f"Updating {expected_count:,} REDCap metadata for {self}")
+            result = self._fetch("metadata", parameters)
+
+            updated_count = result
```

Comment:
Can you confirm that you tested that the API response is a bare number and not `{"count":123}` like the response for record updating?

Reply:
Confirmed. Apparently the REDCap API is inconsistent regarding what it means when we specify `{return: "count"}` in the request.